documentation/book/assembly-kafka-mirror-maker-groupid.adoc

```diff
@@ -0,0 +1,17 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-mirror-maker.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-kafka-mirror-maker-groupid-{context}']
+
+= Consumer group identifier
+
+The Kafka Mirror Maker has a consumer part behaving as any other Kafka consumer client.
+It is in charge to consume the messages from the source Kafka cluster which will be mirrored to the target Kafka cluster.
+The consumer needs to be part of a "consumer group" for being assigned partitions.
```

Comment:
Typography question, possibly for @ncbaratta or @dmuntima: Do we use "scare quotes" like this when alluding to concepts defined elsewhere, or should we use _italics_?